[appservice] "Test Deb Packages Focal" intermittently fails on `test_devops_build_commands_thru_mock.py`

**Describe the bug**

"Test Deb Packages Focal" intermittently fails on `test_devops_build_commands_thru_mock`:

https://dev.azure.com/azure-sdk/public/_build/results?buildId=1133402&view=logs&j=ad51cf76-b294-5bde-777e-be77cc5f3050&t=0e02ab23-4098-514d-f7b9-ab422251bbb2&l=2274

```
==================================== ERRORS ====================================
____ ERROR collecting tests/latest/test_devops_build_commands_thru_mock.py _____
/opt/az/lib/python3.6/site-packages/azure/cli/command_modules/appservice/tests/latest/test_devops_build_commands_thru_mock.py:10: in <module>
    from azure.cli.command_modules.appservice.azure_devops_build_interactive import (
/opt/az/lib/python3.6/site-packages/azure/cli/command_modules/appservice/azure_devops_build_interactive.py:31: in <module>
    from .azure_devops_build_provider import AzureDevopsBuildProvider
/opt/az/lib/python3.6/site-packages/azure/cli/command_modules/appservice/azure_devops_build_provider.py:7: in <module>
    from azure_functions_devops_build.project.project_manager import ProjectManager
/opt/az/lib/python3.6/site-packages/azure_functions_devops_build/project/project_manager.py:14: in <module>
    from ..base.base_manager import BaseManager
/opt/az/lib/python3.6/site-packages/azure_functions_devops_build/base/base_manager.py:6: in <module>
    from vsts.vss_connection import VssConnection
/opt/az/lib/python3.6/site-packages/vsts/vss_connection.py:9: in <module>
    from ._file_cache import RESOURCE_CACHE as RESOURCE_FILE_CACHE
/opt/az/lib/python3.6/site-packages/vsts/_file_cache.py:122: in <module>
    DEFAULT_CACHE_DIR = get_cache_dir()
/opt/az/lib/python3.6/site-packages/vsts/_file_cache.py:117: in get_cache_dir
    os.makedirs(vsts_cache_dir)
/opt/az/lib/python3.6/os.py:220: in makedirs
    mkdir(name, mode)
E   FileExistsError: [Errno 17] File exists: '/root/.vsts/python-sdk/cache'
---------- generated xml file: /azure_cli_test_result/appservice.xml -----------
=========================== short test summary info ============================
ERROR tests/latest/test_devops_build_commands_thru_mock.py - FileExistsError:...

```

Log: [419.log](https://github.com/Azure/azure-cli/files/7309272/419.log)

This is due to a bug of `vsts` that it can't run correctly and fails when racing condition happens. The fix for Yum was made in #18534.

--- a/scripts/release/debian/test_deb_package.py
+++ b/scripts/release/debian/test_deb_package.py
@@ -14,7 +14,7 @@ pytest_base_cmd = '/opt/az/bin/python3 -m pytest -x -v --boxed -p no:warnings --
 pytest_parallel_cmd = '{} -n auto'.format(pytest_base_cmd)
 
 for mod_name in mod_list:
-    if mod_name in ['botservice', 'network', 'cloud']:
+    if mod_name in ['botservice', 'network', 'cloud', 'appservice']:
         exit_code = subprocess.call(['{} --junit-xml /azure_cli_test_result/{}.xml --pyargs azure.cli.command_modules.{}'.format(pytest_base_cmd, mod_name, mod_name)], shell=True)
     else:
         exit_code = subprocess.call(['{} --junit-xml /azure_cli_test_result/{}.xml --pyargs azure.cli.command_modules.{}'.format(pytest_parallel_cmd, mod_name, mod_name)], shell=True)

